content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+Each server software has its own uses and features, so it's important to choose the right one for your needs.
+
+<Callout type="info" title="Note">
+  Server Software(s) are commonly referred to as "Server Jar(s)". This is
+  because server softwares are usually distributed as a `.jar` file.
+</Callout>
+
+# Backend Servers
+
+Backend servers are the most common type of server software.
+There are an incomprehensible amount of backend servers available, however this section will cover the most recommended, stable and widely-used softwares.
+
+<Callout type="info" title="Note">
+  A "fork" is a copy of a software project that is modified in some way.\
+  They typically share the same codebase but have modified (or additional) features and optimizations.\
+  An example of "forks" include: Bukkit -> CraftBukkit -> Spigot -> Paper -> Pufferfish -> etc.
+</Callout>
+
+## Vanilla
+
+Vanilla is the official server software for Minecraft. As the name implies, it is the most basic server software available and has no modifications whatsoever.\
+It is ideal for those who wish to host a small server for a couple of friends or family, **without** altering any mechanics.\
+This server software is **not** recommended for larger servers.
+
+<Card href="https://jar.setup.md/download/vanilla/latest" title="Download">
+  Download a copy of the latest Vanilla server software via jar.setup.md.
+</Card>
+
+## Paper
+
+[Paper](https://papermc.io/) is arguably the most widely used server software today. It is a fork of the Spigot project (with its own API), featuring **numerous optimizations** alongside numerous bug fixes, making it the main choice for many aspiring server owners.
+
+Additionally, it has support for plugins which bring new features, QOL improvements and other enhancements to the game.\
+You can find plugins on platforms such as [SpigotMC](https://www.spigotmc.org/resources/categories/spigot.4/), [Modrinth](https://modrinth.com/plugins), [Hangar](https://hangar.papermc.io/), etc.
+
+<Callout type="warn" title="Warning">
+  While Paper features many optimizations and features, many of these affect vanilla gameplay mechanics.\
+  This makes it unsuitable for servers that wish to maintain a vanilla experience or those who make complex farms that utilize intricate vanilla mechanics.
+</Callout>
+
+<Card href="https://jar.setup.md/download/paper/latest/latest" title="Download">
+  Download a copy of the latest PaperMC server software via
+  [jar.setup.md](https://jar.setup.md/).
+</Card>
+
+## Pufferfish (recommended)
+
+Pufferfish is a [fork](#backend-servers) of [Paper](#paper) that adds significant optimizations to the game without much impact to normal gameplay.\
+These optimizations include entity, hopper, SIMD and other performance improvements.\
+This is highly recommended for all servers due to the significant performance improvements it brings, without affecting plugin compatibility.
```

Comment:
This makes it sound like other Bukkit forks affect plugin compatibility, when this is not true.

From reading your Pufferfish section, I can tell it's your favorite, but knowing what I do about these forks, I'm not sure why.

Reply:
Good point - Resolved in [3e7f1bf](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/3e7f1bf31077e6f84d4e0a49c481a0659798c562)